templates/includes/sidebar.html

```diff
@@ -1,4 +1,10 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">User: {{ user.username }}</a></li>
```

Comment:
The URL for 'taxi:driver-detail' uses 'pk=user.id', assuming that the user's ID corresponds to a driver's ID. Ensure that this assumption is correct, as it may cause issues if the user model is not directly linked to the Driver model.